client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -35,6 +37,20 @@ const HeaderTitle: React.FC<{
   );
 };
 
+// Use dynamic effective user as a mechanism to know whether the system is private or public
+const getScheme = (storageSystem: TTapisSystem): string => {
+  return storageSystem.isDynamicEffectiveUser ? 'private' : 'public';
+};
+
+const getPath = (
+  storageSystem: TTapisSystem,
+  user: TUser | undefined
+): string => {
+  return storageSystem.isDynamicEffectiveUser
+    ? encodeURIComponent('/' + user?.username)
```

Comment:
I wonder if there could be cases in the future where `isDynamicEffectiveUser` is not a good enough flag to assume a child dir with the user's username is present. Perhaps we should add another field to `notes` that's `isMyData` to denote that a username child dir is expected

Reply:
Adjusted the logic

Reply:
Re-tested it.